docs/source/get_involved/code_of_conduct.rst

```diff
@@ -133,13 +163,21 @@ project event or platform include:
 -  nonconsensual or unwelcome physical contact
 -  sustained disruption of talks, events or communications
 -  insults or put downs
--  sexist, racist, homophobic, transphobic, ableist, or exclusionary
-   jokes
+-  sexist, racist, homophobic, transphobic, ableist, or exclusionary jokes
 -  excessive swearing
 -  incitement to violence, suicide, or self-harm
 -  continuing to initiate interaction (including photography or
    recording) with someone after being asked to stop
 -  publication of private communication without consent
+-  decision making on resource allocation which has the effect of excluding people
+   on the basis of protected characteristics, or other forms of systemic discrimination
+-  decision making on resource allocation that confers advantages to personal networks,
+   and excludes others, e.g., preventing advertising of opportunities, not applying best practice in recruitment
+-  reducing transparency of operations of sktime, e.g., by obfuscation of the paper trail
+-  invoking the code of conduct dishonestly, or as a form of bullying,
```

Comment:
disagreed, ruling should be on the balance of probability, based in turn on circumstantial evidence.

There should be burden on the accused to cooperate with proceedings, e.g., to provide proof for how they have handled money in a case of invoice fraud or similar.

Otherwise, anyone can commit a crime and then feel safe by hiding the evidence.

Reply:
I see what you mean, but in line with "assume best intentions" our basic assumption should be that there's been no violation until proven so, and that there should be evidence to prove the wrongdoing proportional to the accusation. Agree that the person who's accused of sktime funds misappropriation for example should have documentation on these funds handling and provide it. 
But we can't just have people going around accusing other community members of any CoC violation they can think about without any facts to back it up. If it's just "my words against your words" then we look at the circumstantial evidence, like what are parties' motivations, history, etc, agree of course, but if there's no substantial proof (circumstantial evidence and motivation counts as proof) - then there's a false accusation, which is also inappropriate and a form of wrongdoing. I thought that's partially what's meant under the "invoking the CoC dishonestly"?

Reply:
>  our basic assumption should be that there's been no violation until proven so, and that there should be evidence to prove the wrongdoing proportional to the accusation. 

fully agreed - all I'm saying, the evidence should be "on the balance of probability", which is a lower standard but common in investigations, say, of research misconduct where the investigating party does not have the rights a judge or the police might have.

See as an example the [UCL guidelines to investigate research misconduct](https://www.ucl.ac.uk/governance-compliance/research-governance), this and other similar documents put the standard of evidence as "on the balance of probabilities".

> there should be evidence to prove the wrongdoing proportional to the accusation. 

That's what it basically means.

> but if there's no substantial proof (circumstantial evidence and motivation counts as proof) - then there's a false accusation, which is also inappropriate and a form of wrongdoing.

The typical standard that justifies making an accusation is typically that it the suspicion is plausible, given the information that the accuser has. It requires no collection of evidence on the side of the accuser.

E.g., to put someone under the suspicion of financial misconduct it suffices if, say, balance sheets are not published in line with institutional expectations. Whereas, for a conviction there must be evidence that the balance sheets were doctored - but the investigation panel has the powers to obtain the balance sheets, or to request them.

> I thought that's partially what's meant under the "invoking the CoC dishonestly"?

Yes, correct. The standard for proving that the CoC has been invoked dishonestly is the same, on the balance of probabilities.

Reply:
To answer the implicit point you are making, I don't think the proposed changes would allow accusation or investigation based on hearsay or made up grounds.

Because the standard of evidence is explicitly set to "on the balance of probabilities".

Reply:
Regarding the burden of proof, I don't think it should lie with the accuser.

I think the process should be such: the CoC committee determines whether the accusation is plausible, then possible immediate actions. If the CoC committee thinks the accusation is plausible, they move to investigate.

It should not be the responsibility of the accuser to investigate or produce proof.

For instance, let's assume a participant of an `sktime` event says they were sexually harassed at an `sktime` event. Things like interviewing witnesses, getting security camera recordings etc are not the burden of the person that was harassed.

It would be similar if the police gets involved - it is the police that investigates first, not the person that was harassed.

Reply:
Same point in a bullying context, made coincidentally by a recent article in HBR:
https://hbr.org/2022/11/how-bullying-manifests-at-work-and-how-to-stop-it?utm_source=pocket-newtab-global-en-GB

Citing:

> Placing the burden of proof and anti-bullying work on the target ignores the fact that bullying is trauma, and that for most people, documenting their own trauma while it is happening while maintaining productivity is an impossible demand

Reply:
(also makes very similar points about resource allocation)

Reply:
What I am saying is that CoC shouldn't accept "bare words" as enough to condemn the person who was accused. Z can say that X is harassing them and not provide any proof because "well you know that was very traumatic event for me", hence the CoC has to simply accept that Z is telling the truth? 
Compare this with Z showing messages / providing a reason why they don't have sufficient proof like "we were at B location on the conference where there were nobody else but me and X, X behaved in ABC manner and told me DEF, and I have shared my concerns with my friend W right after the incident, you can talk to them". 
If there's something serious then the police should be involved and we should wait for the results of the investigation.
I stand by the statement that "It is better that ten guilty persons escape than that one innocent suffer".